pom.xml

```diff
@@ -9,24 +9,22 @@
   
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.bdgenomics.adam</groupId>
-  <artifactId>adam-parent</artifactId>
+  <artifactId>adam-parent_2.10</artifactId>
   <version>0.16.1-SNAPSHOT</version>
   <packaging>pom</packaging>
-  <name>ADAM</name>
+  <name>ADAM_2.10</name>
   <description>A fast, scalable genome analysis system</description>
   <url>http://bdgenomics.org/</url>
 
   <properties>
     <java.version>1.7</java.version>
-    <scala.version>2.10.4</scala.version>
```

Comment:
So, Maven doesn't _strictly_ disallow variables in the `<artifactId>` field (it just gives you a warning that "this might break things"), but then I did https://github.com/bigdatagenomics/utils/commit/47e325a6558b512108d5e695b33667cdd9d6ceae, and tried to cut a release of that.

Let's just say it broke in spectacular and vengeful ways. After I made that commit, I spent a long time wondering asking reflecting on the mistakes in my life that led me to write `47e325a`. The commit seemed so innocent at the time, but alas, my youthful ignorance (of mvn) led me astray.